x-pack/test/apm_api_integration/tests/service_groups/save_service_group.spec.ts

```diff
@@ -65,12 +65,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     afterEach(deleteServiceGroups);
 
     it('creates a new service group', async () => {
-      const response = await callApi({
+      const serviceGroup = {
         groupName: 'synthbeans',
         kuery: 'service.name: synth*',
-      });
+      };
+      const response = await callApi(serviceGroup);
```

Comment:
Would be nice if this was more clear:

```suggestion
      const response = await createServiceGroupApi(serviceGroup);
```